app/dashboard/helpers.py

```diff
@@ -867,6 +880,11 @@ def process_bounty_changes(old_bounty, new_bounty):
     if new_bounty.fulfillments.exists():
         profile_pairs = build_profile_pairs(new_bounty)
 
+    # Send an Email if this is a LowBall bounty
+    if(not old_bounty or old_bounty.value_in_usdt != new_bounty.value_in_usdt):
+        if is_lowball_bounty(new_bounty.value_in_usdt):
```

Comment:
I have just a question about this, what happens if is a token and for some reason we don't have the usd value? It will affect the rest of the flow or just fail silently?

Reply:
@octavioamu thats a really good point - @oddballbutler can you please add a test case to cover this?